func-complete-check/check-receiver/index.ts

```diff
@@ -0,0 +1,30 @@
+import { AzureFunction, Context } from '@azure/functions'
+import * as schemas from '../typings/message-schemas'
+import { performance } from 'perf_hooks'
+const functionName = 'check-receiver'
+import V3 from './v3'
+
+const queueTrigger: AzureFunction = async function (context: Context, completedCheck: schemas.SubmittedCheckMessageV3): Promise<void> {
+  const start = performance.now()
+  const version = completedCheck.version
+  context.log(typeof(version))

```

Comment:
Looks like incomplete logging

Reply:
this was done to understand why version needed to be a string, will remove.